pkgs/top-level/config.nix

```diff
@@ -137,6 +137,17 @@ let
       '';
     };
 
+    allowVariants = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Whether to expose the nixpkgs variants.
+
+        Variants are subsets of the current nixpkgs instance with different options applied.
```

Comment:
They're not really subsets. They contain just as much as the original set.

Reply:
What's a better way to explaining it?

Reply:
They're differently instantiated package sets.

Reply:
I've reworded things slightly, idk if that's any better.